### PR TITLE
feat: add tipFormatter for Slider/Range with tooltip, close: #232

### PR DESCRIPTION
**Pull request by benjycui**
close: #232

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ const Range = createSliderWithTooltip(Slider.Range);
 
 [Online demo](http://react-component.github.io/slider/examples/handle.html)
 
+After Range or Slider was wrapped by createSliderWithTooltip, it will have the following props:
+
+| Name         | Type    | Default | Description |
+| ------------ | ------- | ------- | ----------- |
+| tipFormatter | (value: number): React.ReactNode | value => value | A function to format tooltip overlay, if it's null, tooltip will be invisible |
```
Comment (paranoidjk) [inline README.md]: >  if it's null, tooltip will be invisible

This logic seems kind of weird，If one use `createSliderWithTooltip`, I think he wants to show tooltip.

`tipFormatter` is  just a way of data transform which default value is not transform

Comment (benjycui) [inline README.md]: Yep, I just use prev logic...

**Comment by benjycui** [comment]
Updated

**Comment by paranoidjk** [comment]
👍 +1

**Comment by benjycui** [comment]
@paranoidjk so, just merge it.. and publish

**Comment by paranoidjk** [comment]
@benjycui Coverage decreased

**Comment by benjycui** [comment]
😢 it's not easy to test thing like tooltip, just ignore it...

**Comment by paranoidjk** [comment]
It's a little strange, you just add one single api, and Coverage decreased `(-4.5%)`, no need to worry why?

**Comment by paranoidjk** [comment]
6.2.0

**Comment by benjycui** [comment]
Possible reason: https://github.com/nickmerwin/node-coveralls/issues/158

**Pull request merged**